### ISSUE TITLE
Docs/Tutorial/Demo of Cards

### ISSUE THREAD

**Issue by augiwan** [florent37/MaterialViewPager#203]
Please could you add some helpful pointers regarding customizing the card views, for example, adding buttons and setting click listeners for buttons on the card. Most of the available documentation online to set click listeners uses a ViewHolder implementation, but the sample code here utilizes an Adapter implementation. It would really help if you showed some examples. :smile:

**Comment by PedroBarbosaPT** [comment]
Have you find anything?
Want that too.

**Comment by bloder** [comment]
What's exactly your question about it? I can't show you my example but i can answer you what you want.

**Comment by augiwan** [comment]
Okay, for example, could you showcase a view with two tabs using MaterialViewPager, containing a few cards in each tab, having two buttons inside each CardView and a click listener attached to each button.

**Comment by bloder** [comment]
Look this - https://github.com/bloder/megan-fox-layout

but cards doesn't have buttons with click event, I can improve it in project but it's easy do it, you just need implement click listener in your card adapter, in onBindViewHolder you get your button from view holder with that param "holder", and just add the click event, it's very easy. Then in this example you will find the view with... i don't remember... i guess 4 or 5 tabs using MaterialViewPager with i guess 20 cards in each tab, and you will find Megan Fox *------*

**Comment by augiwan** [comment]
Thanks for the reply! Is this based on the MaterialViewPager project?

**Comment by bloder** [comment]
Yes, its a MaterialViewPager example but I just removed those boring images and replaced with Megan fox pictures =D
